lego/apps/companies/serializers.py

```diff
@@ -112,6 +112,7 @@ class Meta:
             "address",
             "logo",
             "thumbnail",
+            "active",
```

Comment:
Is this necessary? The frontend doesn't need to know whether the company is active or not, we just feed it a list of active companies? 

Or do we need it here to be able to filter the queryset in the view? I don't think so?

Reply:
Yeah, it's unnecessary. Removed it now.